src/lib.rs

```diff
@@ -460,29 +546,30 @@ impl<T> Grid<T> {
     }
 
     /// Traverse the grid with row and column indexes.
-    /// 
+    ///
     /// # Examples
-    /// 
+    ///
     /// ```
     /// use grid::*;
     /// let grid: Grid<u8> = grid![[1,2][3,4]];
     /// let mut iter = grid.indexed_iter();
     /// assert_eq!(iter.next(), Some(((0, 0), &1)));
     /// ```
-    /// 
+    ///
     /// Or simply unpack in a `for`  loop:
-    /// 
+    ///
     /// ```
     /// use grid::*;
     /// let grid: Grid<u8> = grid![[1,2][3,4]];
     /// for ((row, col), i) in grid.indexed_iter() {
     ///     println!("value at row {row} and column {col} is: {i}");
     /// }
     /// ```
-    pub fn indexed_iter(&self) -> impl Iterator<Item=((usize, usize), &T)> {
-        self.data.iter().enumerate().map(move |(idx, i)| 
-            ((idx / self.cols, idx % self.cols), i)
-        )
+    pub fn indexed_iter(&self) -> impl Iterator<Item = ((usize, usize), &T)> {
```

Comment:
Why did you change this method? How does it relate to the pull request?

Reply:
I didn't mean to change it, it looks like rustfmt did that when I saved the file. I can undo the change if it's an issue.

Reply:
Ah right. It is only formatting. Fine